governance.md

```diff
@@ -0,0 +1,150 @@
+
+## Proposals
+Proposals can vary broadly in their stated goals and requests. At this time proposals can be made to the ODC community or to the ODC Jedi through the Governance Repository Proposal form. Each new proposal will get it's own Discord thread under the Proposals channel. 
+
+- They may ask the community to endorse a specific policy and procedure (e.g. “Should ODC allow VC backed projects?”), or ask the community to build something (e.g. “Building a new software tool” or “Writing improved documentation”).
+- They may have no funding requests attached (e.g. “Should we adopt a new Foundation for the DAO”), or make an explicit request for funding (e.g. “We would like to create a working group focused on building a decentralized architecture for Gitcoin Grants”).
+
+After a period of time the participants in the Discord thread can ask for a vote to be taken on the proposal.  
+
+## Elections and Voting
+At this time, only Working Groups and Projects elect leads by using the ODC voting tooling. However, the various ODC groups within the community can use the ODC voting tooling at any time. 
+
+In the near future, ODC decisions and voting will be done on-chain. The ODC Jedi are not ready to operationally support this approach yet. 
+
+Find more information about community-organized [Voting here](TBD).
+
+## Funding
+At this time funding projects and work activities is decided by the ODC Jedi. 
+
+# Committee
+
+## OpenData Community Jedi
+Original founding members of the OpenData Community (ODC). As ODC grows, the ODC Jedi will be replaced with a governing board of sponsors, partners, and elected members. 
+
+Requirements:
+- Be an ODC member in good standing
+- Elected by the Working Group team members
+
+Benefits:
+- All governance, business development, finance, and engineering decisions initially fall to it's members
+
+# Groups
+
+## ODC Projects
+Achieving this stage proves that the team practices the basics of agile software development. ODC Projects are the projects under the governance of the ODC. ODC projects are listed in the [ODC Projects page] and their repositories can be found within the [ODC Github Organization]. It is not mandatory for a Sandbox team to become an official ODC project. The [ODC Projects] are groups that are directly responsible for the production of ODC software releases.
```

Comment:
I don't like the term "ODC Projects"

We should consider the experience of a project founder.
They want the support of the ODC however they do not want to turn over everything to the ODC and just walk away - and neither do we!

These are not ODC projects - they remain their own project in many cases